src/commands/utils/udfRegistration.ts

```diff
@@ -0,0 +1,43 @@
+import { Uri, window } from "vscode";
+import { Logger } from "../../logging";
+import { inspectJarClasses, JarClassInfo } from "../../utils/jarInspector";
+
+const logger = new Logger("commands.flinkUDFs/udfRegistration");
```

Comment:
Same super-nit as the parent branch though 😅 but for accuracy based on the filename:
```suggestion
const logger = new Logger("commands.utils.udfRegistration");
```

Reply:
thanks I was meaning to ask you about that and test names conventions